united-fare-finder.py

```diff
@@ -36,45 +36,96 @@
 elem = driver.find_element(By.ID, "closeBtn")
 elem.click()
 
-elem = driver.find_element(By.ID, "bookFlightOriginInput")
-elem.clear()
-elem.send_keys("ORD")
-elem.send_keys(Keys.TAB)
-
-elem = driver.find_element(By.ID, "bookFlightDestinationInput")
-elem.clear()
-elem.send_keys("HNL")
-elem.send_keys(Keys.TAB)
-
-elem = driver.find_element(By.ID, "DepartDate")
-elem.clear()
-elem.send_keys("03/03/23")
-elem.send_keys(Keys.TAB)
-
-elem = driver.find_element(By.ID, "ReturnDate")
-elem.clear()
-elem.send_keys("03/10/23")
-elem.send_keys(Keys.TAB)
-
-elem = driver.find_element(
-  By.XPATH, '//*[@id="bookFlightForm"]/div[1]/div[1]/label')
-elem.click()
-
-elem = driver.find_element(
-  By.CSS_SELECTOR,
-  "button.app-components-BookFlightForm-bookFlightForm__findFlightBtn--1lbFe")
-elem.submit()
-
-time.sleep(10)
-
-# scrape fare data
-page_source = driver.page_source
-flight_elements = parsePageSource(page_source)
-
-# write to file
-for i in range(len(flight_elements)):
-    for j in range(len(flight_elements[i])):
-        print(*flight_elements[i][j])
-    print()
+origin = "ORD"
+dest = [
+  "hnl", "slc", "dfw", "lax", "sea", "atl",
+  "mia", "mco", "sfo", "jfk", "ewr", "lga", "dca", "iad"
+]
+depDate = "03/01/2023"
+retDate = "03/10/2023"
```

Comment:
its common to name constatnts in all caps (and snake cased) at the beginning of the file

Reply:
done